### DISCUSSION TITLE
Is there a way to move the Key visualizer?

### DISCUSSION THREAD

**Discussion by BrandonDunlevy** [mulaRahul/keyviz#247] [General]
I am trying to move the window that shows the keys pressed to a different monitor or even more center on my monitor is there a way to do that?

**Comment by venomgfx** [comment]
Unfortunately it seems the project is not actively maintained anymore. 

Even without UI, a command line solution like `keyviz.exe --display 2` would be amazing.

**Comment by mulaRahul** [comment]
Multi-monitor support has been added with [v2.0.0a2](https://github.com/mulaRahul/keyviz/releases/tag/v2.0.0a2)
